android/titanium/src/java/org/appcelerator/titanium/view/TiUIView.java

```diff
@@ -895,21 +895,41 @@ public void propertyChanged(String key, Object oldValue, Object newValue, KrollP
 				ViewCompat.setElevation(nativeView, TiConvert.toFloat(newValue));
 			}
 		} else if (key.equals(TiC.PROPERTY_TRANSLATION_X)) {
-			if (nativeView != null) {
-				ViewCompat.setTranslationX(nativeView, TiConvert.toFloat(newValue));
+			if (getOuterView() != null) {
```

Comment:
In my PR I just tested the transformation and didn't test the elevation part. But in my test I found out that nativeView was moving the view inside the borderView complex. I found the function https://github.com/m1ga/titanium_mobile/blob/8d5e2d9e28fdd7c57b097e240d8a76e92906d258/android/titanium/src/java/org/appcelerator/titanium/view/TiUIView.java#L1430 that will take the borderView or nativeView so I thought it would be good to use in my cases so it'll use the proper view. I can test the other cases too with a border view